src/Registry/Registry.jl

```diff
@@ -362,7 +362,8 @@ function update(regs::Vector{RegistrySpec} = RegistrySpec[]; io::IO=stderr_f(),
                             try
                                 download_verify(url, nothing, tmp)
                             catch err
-                                @error "could not download $url" exception=err
+                                push!(errors, (reg.path, "failed to download from $(url). Exception: $(sprint(showerror, err))"))
+                                @goto done
```

Comment:
Are `@goto`s scoped? There is `@goto` with the same label later in this function.

Reply:
Yeah I found out through this that they are. I could use different labels to make it less confusing though